src/i18n/en.toml

```diff
@@ -392,6 +392,9 @@ titleRecentChallenges = "Recent completed challenges"
 titleResults = "Results"
 titleUpcomingChallenges = "Upcoming challenges"
 titleYourResultsSince = "Your results since {date}"
+tabConsistency = "Consistency"
```

Comment:
Pravidelnost -> Frequency

Look at old DPNK app code trans strings please.

Reply:
In this context, is it better to use `regularity`?
See https://github.com/auto-mat/do-prace-na-kole/blob/master/apps/dpnk/locale/en/LC_MESSAGES/django.po#L4370

Reply:
Ok. Yes are right.